readme.md

```diff
@@ -254,6 +254,21 @@ Yes please!
 
 `DEBUG=0x* 0x my-app.js`
 
+## Docker
+In a docker container, due to security reasons, you most likelly gonna end up with the follwing error:
+
```

Comment:
1) let's move this up to just beneath "Generating"
2) can we keep the tone consistent (e.g. we, instead of you, avoid colloqualisms) with the rest of the readme, also - typos: 
Due to security reasons Docker containers tend to result in the following error: 

We can work around this problem by running our container with the `--privileged` option, or add `privileged: true` in our `docker-compose.yml` file. See the [Docker docs](https://docs.docker.com/engine/reference/run/#runtime-privilege-and-linux-capabilities) for more info